frameworks/keyed/mikado/src/main.js

```diff
@@ -0,0 +1,38 @@
+import Mikado from "../node_modules/mikado/src/mikado.js";
+import { route } from "../node_modules/mikado/src/event.js";
+import tpl_app from "./template/app.js";
+import tpl_item from "./template/item.js";
+import buildData from "./data.js";
+
+Mikado.once(document.body, tpl_app).eventCache = true;
+
+// This implementation is using a full declarative paradigm,
```

Comment:
not sure this is true here

Reply:
This is a declarative usage:

```js
view.update(1, data);
view.add(data);
view.remove(data);
```

This isn't:

```js
const data = [];
view.render(data);
```